### PR TITLE
improve header

### PR DESCRIPTION
**Pull request by VKarpina**

### CODE REVIEW

File: src/css/header.css [hunk 1]
```diff
@@ -47,17 +42,21 @@
   display: none;
 }
 
+@media screen and (min-width: 375px) {
+  .header-container {
+    width: 335px;
+  }
+}
```
Comment (mikk1max) [inline src/css/header.css]: delete this media rule, it must be rubber layout, but not adaptive

File: src/css/header.css [hunk 2]
```diff
@@ -1,22 +1,20 @@
 /** ========= HEADER ========= */
-
-* {
-  box-sizing: border-box;
-}
-
 .header-container {
   position: absolute;
-  width: 335px;
+  min-width: 280px;
```
Comment (mikk1max) [inline src/css/header.css]: add here max-width: 335px, and width: 80vw, then it will work correctly

File: src/css/header.css [hunk 3]
```diff
@@ -47,17 +42,21 @@
...
+    width: 335px;
+  }
+}
+
 @media screen and (min-width: 768px) {
   .header-container {
     width: 704px;
-    padding: 12px 28px;
+    height: 48px;
+    padding: 0 28px;
```
Comment (mikk1max) [inline src/css/header.css]: padding: 4px 28px

File: src/css/header.css [hunk 4]
```diff
@@ -68,65 +67,59 @@
...
 
 @media screen and (min-width: 1280px) {
   .header-container {
     width: 1080px;
-    padding: 12px 30px;
-  }
-
-  .header-logo {
-    margin-right: 296px;
+    padding: 0 30px;
```
Comment (mikk1max) [inline src/css/header.css]: padding: 4px 30px

**Review by mikk1max — COMMENTED** [review]
You have to make some fixes, but in general very cool!

File: src/css/header.css (hunk 1 above)
Comment (VKarpina) [reply src/css/header.css:49]: According to our technical task the layout is rubber from 320px and becomes adaptive from 375px.

**Review by mikk1max — COMMENTED** [review]
All the points were described in the videos I sent you. Good job!

**Comment by mikk1max** [comment]
Very nice, I will merge it later!

**Comment by mikk1max** [comment]
We choose a better solution

**Pull request closed without merging**